questions/conceptual.md

```diff
@@ -0,0 +1,59 @@
+## Conceptual Exercise
+
+### Answer the following questions below:
+
+**1. What is PostgreSQL?**  
+A highly stable open source object relational database management system, first released in 1996
+
+**2. What is the difference between SQL and PostgreSQL?**  
+PostgreSQL is an advanced version of SQL providing support to different functions of SQL like foreign  
+keys, subqueries, triggers, and different user-defined types and functions. In general PostgreSQL is  
+known for having additional robust features over SQL.
+
+**3. In `psql`, how do you connect to a database?**  
+psql db_name or \c db_name
+
+**4. What is the difference between `HAVING` and `WHERE`?**  
+WHERE is used to filter records before any groupings take place. HAVING is used to filter values after  
+they have been groups. Only columns or expressions in the group can be included in the HAVING clause's  
+conditions
+
+**5. What is the difference between an `INNER` and `OUTER` join?**  
+The major difference between INNER and OUTER joins is that INNER joins result in the intersection of  
+two tables, whereas OUTER joins result in the union of two tables.  In other words - INNER join returns  
+only the rows that match a given condition in both tables whereas OUTER joins returns all rows of the  
+two tables.

```

Comment:
What you have re: `OUTER` joins in #6 is correct. Only `FULL JOIN` will get you all rows of both tables (matched where possible).